packages/block-library/src/cover/index.php

```diff
@@ -47,18 +47,14 @@ function render_block_core_cover( $attributes, $content ) {
 		}
 		$current_featured_image = get_the_post_thumbnail_url();
 
-		$styles = 'background-image:url(' . esc_url( $current_featured_image ) . '); ';
-
-		if ( isset( $attributes['minHeight'] ) ) {
-			$height_unit = empty( $attributes['minHeightUnit'] ) ? 'px' : $attributes['minHeightUnit'];
-			$height      = " min-height:{$attributes['minHeight']}{$height_unit}";
-
-			$styles .= $height;
-		}
-
 		$processor = new WP_HTML_Tag_Processor( $content );
 		$processor->next_tag();
-		$processor->set_attribute( 'style', $styles );
+
+		$styles         = $processor->get_attribute( 'style' );
+		$merged_styles  = ! empty( $styles ) ? $styles . ';' : '';
+		$merged_styles .= 'background-image:url(' . esc_url( $current_featured_image ) . ');';
```

Comment:
```suggestion
		$styles        = $processor->get_attribute( 'style' );
		$merged_styles = ! empty( $styles ) ? $styles . ';' : '';

		if ( $current_featured_image ) {
			$merged_styles .= 'background-image:url(' . esc_url( $current_featured_image ) . ');';
		}
```

Maybe this should be done as a follow-up PR, but how about preventing the `background-image` property with an empty url from being output when there is no post featured image?

![empty-url](https://user-images.githubusercontent.com/54422211/228729226-aaa213f5-ff79-4e7e-9c4c-911567e1dd83.png)

Reply:
Right. Let's do this in a follow-up 👍

Reply:
Actually, there's no need for that. You can't enabled a fixed background if if the featured image isn't set. So this code is never executed.

Reply:
I think the following steps could happen:

- Add the featured image to the post from the Post panel
- Insert a cover block
- Enable "Fixed background" or "Repeated background"
- Delete the featured image from the Post panel

After this procedure, the media settings will not appear in the cover block, but the `hasParallax` or `isRepeated` attributes will be retained.

Reply:
Good point. Never trust the user 😄

I've changed on a local branch and will push it shortly.